mix.exs

```diff
@@ -17,15 +17,15 @@ defmodule PhoenixHtmlSanitizer.Mixfile do
       ],
       build_embedded: Mix.env() == :prod,
       start_permanent: Mix.env() == :prod,
-      deps: deps
+      deps: deps()
     ]
   end
 
   # Configuration for the OTP application
   #
   # Type `mix help compile.app` for more information
   def application do
-    [applications: [:logger, :html_sanitize_ex]]
+    [extra_applications: [:logger, :html_sanitize_ex]]
```

Comment:
I was running into a very odd issue.

If I completely delete the build and _deps folders, it will compile and run the tests without issue.

But when I run the tests again, I see test failures like this:

```
  1) test using a different sanitizer mode (PhoenixHtmlSanitizerViewTest)
     test/phoenix_html_sanitizer_test.exs:14
     ** (UndefinedFunctionError) function Phoenix.HTML.raw/1 is undefined (module Phoenix.HTML is not available)
```

Looks like an applications vs. extra_applications issue. Changing this looks to have solved it.